App/src/pages/RegularGroups.tsx

```diff
@@ -0,0 +1,83 @@
+import { Container, Table, Thead, Tbody, Tr, Th, Td } from "@chakra-ui/react";
+import { useContext, useEffect, useState } from "react";
+import { GetAllGroups, GroupDTO, MultiGroupResponseDTO } from "../services/group";
+import { UserContext } from "../App";
+import { UserDTO } from "../services/user";
+import Header from "../shared/Header";
+
+export default function DisplayGroups() {
+    const [myGroups, setMyGroups] = useState<GroupDTO[]>([]);
+    const user = useContext(UserContext);
+    const userDTO: UserDTO = { firstName: user.firstName, id: "7", lastName: user.lastName, email: user.email };
+    useEffect(() => {
+        AllGroups();
+        console.log(user);
+    }, []);
+
+    function AllGroups() {
+        GetAllGroups()
+            .then(
+                (res: MultiGroupResponseDTO) => {
+                    setMyGroups(res.data);
+                    setMyGroups(res.data.filter(membership));
+                },
+                (rej) => {
+                    console.log(rej);
+                }
+            )
+            .catch((err) => {
+                console.log(err);
+            });
+    }
+    function membership(GroupDTO: GroupDTO): boolean {
```

Comment:
The function should be named something more descriptive like isMemberOfGroup

Reply:
Done